### PR TITLE
Fix lvmo deployment

### PR DESCRIPTION
**Pull request by AviadP**
add check for all containers statuses in lvm-operator pod before creation of lvm-clusrer

### CODE REVIEW

File: ocs_ci/ocs/resources/pod.py [hunk 1]
```diff
@@ -2015,6 +2015,62 @@ def check_pods_in_running_state(
...
+        for containers in TimeoutSampler(
+            timeout=timeout,
+            sleep=sleep,
+            func=check_containers_are_running,
+            namespace=namespace,
+            pod_names=pod_name,
+        ):
+            # Check if all the pods in running state
+            if containers:
+                logger.info("All the pods reached status running!")
```
Comment (ebenahar) [inline ocs_ci/ocs/resources/pod.py:2064]: replace with "All the containers reached status running!"

File: ocs_ci/ocs/resources/pod.py [hunk 2]
```diff
@@ -2015,6 +2015,62 @@ def check_pods_in_running_state(
...
+                return True
+
+    except TimeoutExpiredError:
+        logger.error(
+            f"Not all the containers reached status running " f"after {timeout} seconds"
```
Comment (ebenahar) [inline ocs_ci/ocs/resources/pod.py:2069]: f"Not all the containers reached status running after {timeout} seconds"

File: ocs_ci/ocs/resources/pod.py (hunk 1 above)
Comment (AviadP) [reply ocs_ci/ocs/resources/pod.py]: fixed

File: ocs_ci/ocs/resources/pod.py (hunk 2 above)
Comment (AviadP) [reply ocs_ci/ocs/resources/pod.py]: fixed

**Comment by AviadP** [comment]
verified by: https://url.corp.redhat.com/f54643b

**Review by ebenahar — APPROVED** [review]

**Comment by petr-balogh** [comment]
@AviadP @shyRozen is this still relevant or fixed in other PR?

**Pull request closed without merging**